diagnostic.md

```diff
@@ -99,7 +120,8 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+color: yellow
+font-family: sans-serif
```

Comment:
Cursive would be the font-family due to inline styling on the class in the HTML.